### ISSUE TITLE
error when installing - cannot find "ggproto"

### ISSUE THREAD

**Issue by countershane** [dkahle/ggmap#77]
<img width="709" alt="screen shot 2015-12-09 at 9 33 20 pm" src="https://cloud.githubusercontent.com/assets/11520828/11705262/9dd9bba8-9ebc-11e5-9f71-826ccfe23a0c.png">

**Comment by dkahle** [comment]
Have you tried loading the dev version of ggplot2?

**Comment by countershane** [comment]
I haven't. My intention is to fork & edit yours so that I can use my dev api from Google, and get past the 2,500 query limit. I was able to hard code it to work before. But I'm trying to make it work and push it for others to use. 
For some reason when I tried making a 3rd userType in a similar fashion to the way the others are made, it didn't work. Also, I've noticed when trying to use a client/signature that it raises the query limit but doesn't append that to the end of the url.
I wasn't able to figure that out :/
